### DISCUSSION TITLE
`_quarto.yml`: No inline code for field content?

### DISCUSSION THREAD

**Discussion by r-cheologist** [quarto-dev/quarto-cli#2384] [Q&A] [answered]
I'm trying the following:
```
website:
  title: "Site"
  page-footer: "`r paste('©', lubridate::year(lubridate::now()), 'Me')`"
```
and only get verbatim content **not** the computed and assembled date string.

Can this be made work in `Quarto`?

**Comment by cderv** [comment, marked as answer]
Just to share that there is an issue about that you can follow: https://github.com/quarto-dev/quarto-cli/issues/1391

But overall, It is not possible to use **knitr** feature like that (inline code) outside of `.qmd` document using knitr.  R is not run when the YAML is parsed.

**Reply by r-cheologist** [reply]
Thanks for pointing that out.Closing this then.
